Userland/Libraries/LibWeb/EntriesAPI/FileSystemEntry.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2024, Jamie Mansfield <jmansfield@cadixdev.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibWeb/Bindings/FileSystemEntryPrototype.h>
+#include <LibWeb/Bindings/Intrinsics.h>
+#include <LibWeb/EntriesAPI/FileSystemEntry.h>
+#include <LibWeb/HTML/Window.h>
+
+namespace Web::EntriesAPI {
+
+JS_DEFINE_ALLOCATOR(FileSystemEntry);
+
+WebIDL::ExceptionOr<JS::NonnullGCPtr<FileSystemEntry>> FileSystemEntry::create(JS::Realm& realm, EntryType entry_type, ByteString name)
```

Comment:
Looks like this can't throw? We can just make the return type `JS::NonnullGCPtr<FileSystemEntry>`

Reply:
Good point, it can't. Resolved :^)